backend/irc/IrcMessageProcessor.ts

```diff
@@ -160,7 +160,7 @@ class IrcMessageProcessor {
   private async processJoin(ircMessage: IrcMessage): Promise<void> {
     const nick = ircMessage.prefix && ircMessage.prefix.split('!')[0];
     if (nick != this.joinConfig.user) {
-      console.log(nick, nick!.slice(1), nick![0] === '@' || '%' || '+');
+      //console.log(nick, nick!.slice(1), nick![0] === '@' || '%' || '+');
```

Comment:
By removing the debug line I didn't mean commenting it out :P

Reply:
That's a different one as well apparently

Reply:
this debug log has been in there for a while and I just caught it. Maybe we would use it in the future (hopeful but improbable). It is removed.